src/components/Parameter.jsx

```
@@ -107,7 +107,7 @@ const useParameterAsyncValidation = ({
   return validator;
 };
 
-const Parameter = ({ parameter, form, toolName }) => {
+const Parameter = ({ parameter, form, toolName, disabled: paramDisabled }) => {
```

ESLint

[error] 'paramDisabled' is defined but never used.

(no-unused-vars)